Adding virtio-scsi controller support for libvirt for install and import using virt-install

sathnaga opened the pull request:
The respective controller was not getting added to the virt-install
command line and due to which the automated installation of virtio-scsi(ppc64) was not
possible, this patch addresses the automatic choosing of virtio-scsi controller
in libvirt when it is chosen as disk type and respectively for other disk types.

Signed-off-by: Satheesh Rajendran <sathnaga@linux.vnet.ibm.com>

waynesun09 commented on shared/cfg/guest-hw.cfg:
```diff
@@ -72,10 +72,12 @@ variants:
         drive_format=scsi-hd
         cd_format=scsi-cd
         scsi_hba=virtio-scsi-pci
+	libvirt_controller=virtio-scsi
```
align the params here, one space needed before 'libvirt_controller'

waynesun09 commented on virttest/libvirt_vm.py:
```diff
@@ -413,6 +413,21 @@ def add_import(help_text):
+
+        def check_controller(virt_install_cmd_line, controller):
+            found = False
+            output = re.findall(r"controller\stype=(\S+),model=(\S+)",virt_install_cmd_line)
```
one space needed after ',', you can check this in travis CI failure log

sathnaga commented:
@waynesun09: pls let me know if everything is fine and pls push it. Thanks in advance.

will-Do commented:
Codes are good, but please squash your commits, thanks

sathnaga commented:
Messed up this  branch during rebase, created another pull request and rebased, squashed commits Adding virtio-scsi controller support for libvirt for install and import... #2044.

Please verify the same and push it, Thanks in advance. :-)
@will-Do @waynesun09 @cevich

waynesun09 commented:
@sathnaga ok, I'll check the new PR, you can close this one now.
